packages/kolibri/utils/sanitizeError.js

```diff
@@ -0,0 +1,54 @@
+import pickBy from 'lodash/pickBy';
+
+const SENSITIVE_FIELDS = ['password', 'token', 'secret', 'apikey', 'api_key'];
+const SENSITIVE_HEADERS = ['authorization', 'x-api-key', 'x-auth-token'];
+const MASK = '********';
+
+function maskObject(obj, sensitiveKeys) {
+  if (!obj || typeof obj !== 'object') return obj;
```

Comment:
I asked myself "what happens when it's not an obj" while reading `_handleData` and this is the perfect answer :+1: